src/user/tests/helpers.py

```diff
@@ -111,6 +110,15 @@ def create_random_authenticated_user_with_reputation(unique_value, reputation):
     return user
 
 
+def create_old_random_authenticated_user_with_reputation(unique_value, reputation):
```

Comment:
Nit: The name can be slightly confusing. Is it worth having an extra helper for this when it's only used once?